src/main/webapp/portal/main/partials/footer.html

```diff
@@ -12,7 +12,7 @@
 				<span><a ng-if='sessionCheckCtrl.back2ClassicURL' ng-href='{{sessionCheckCtrl.back2ClassicURL}}'>MyUW Classic</a></span>
 			</div>
 			<div>
-				<span>{{sessionCheckCtrl.user.version}}</span><span> - </span><span>{{sessionCheckCtrl.user.serverName}}</span><span> - </span><span>{{sessionCheckCtrl.user.sessionKey}}</span></span>
+				<span>{{sessionCheckCtrl.user.version}}</span><span ng-show="sessionCheckCtrl.user.serverName"> - </span><span>{{sessionCheckCtrl.user.serverName}}</span><span ng-show="sessionCheckCtrl.user.sessionKey"> - </span><span>{{sessionCheckCtrl.user.sessionKey}}</span></span>
```

Comment:
Might want My UW folks to weigh in; @jhanstra or @vertein do you have anything looking for these specific values in dedicated spans? They don't have ids or classes, so I'm betting not.

A local test of Paul's technique shows it works.